openapi/openapi.yml

```diff
@@ -74,6 +75,34 @@ paths:
                         application/json:
                             schema:
                                 $ref: '#/components/schemas/Error'
+    /nodes/nodePublicKey/{nodePublicKey}:
+        get:
+            summary: Returns the node information for the given transport public key.
+            tags:
+                - node
+            operationId: getNodeByNodePublicKey
+            parameters:
+                - name: nodePublicKey
+                  required: true
+                  description: The node's transport public key.
+                  in: path
+                  schema:
+                      type: string
+                      pattern: '^([0-9A-Fa-f]){64}$'
+                  example: E3FC28889BDE31406465167F1D9D6A16DCA1FF67A3BABFA5E5A8596478848F78
+            responses:
+                200:
+                    description: The node information for the given transport public key.
```

Comment:
depend what we want to use `nodePublicKey ` or `transportPublicKey`

`transport public key` -> `node public key`